versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/DatabaseAdapterFactory.java

```diff
@@ -33,44 +33,53 @@
  * @param <CONFIG> the configuration-options type used to configure database-adapters produced by
  *     this factory.
  */
-public interface DatabaseAdapterFactory<CONFIG extends DatabaseAdapterConfig<?>> {
-  Builder<CONFIG> newBuilder();
+public interface DatabaseAdapterFactory<CONFIG, CONNECTOR> {
```

Comment:
Why remove the `extends DatabaseAdapterConfig<?>` / why no `extends` for `CONNECTOR`?

Reply:
This is an artifact of my local experiments... will restore

Reply:
type parameters narrowed in c96f9344d07e4ee78f3bd9596af9f5a366a98d47